ProcessList.c

```diff
@@ -626,10 +632,16 @@ void ProcessList_scan(ProcessList* this, bool pauseProcessUpdate) {
 
    ProcessList_goThroughEntries(this, false);
 
+   uint maxUid = 0;
```

Comment:
`uint`? Did you mean `unsigned int`?

Reply:
Prefer to use `uint32` in this case, as we're talking about support for 32 bit UIDs and using the native `uid_t` type wouldn't work in this platform neutral section.

Reply:
> uint? Did you mean unsigned int?

Yep :disappointed_relieved: 

> Prefer to use uint32_t in this case, as we're talking about support for 32 bit UIDs and using the native uid_t type wouldn't work in this platform neutral section.

Looks like `uid_t` is already used here, so I have used that.